webpack/assets/javascripts/foreman_i18n.js

```diff
@@ -0,0 +1,29 @@
+import Jed from 'jed';
+import { deprecateObjectProperty } from './foreman_tools';
+
+const emptyLocales = {
+  en: { domain: 'app', locale_data: { app: { '': {} } } },
+};
+
+const getLocale = () => {
+  const locale = document.getElementsByTagName('html')[0].lang;
+  return locale.length === 0 ? 'en' : locale.replace(/-/g, '_');
+};
+
+const locales = window.locales || emptyLocales;
+const i18n = new Jed(locales[getLocale()]);
```

Comment:
Could we export Jed as i18n.jed or something similar? I'd like to avoid exporting it as default, since there's a concurrent PR, where I'm adding some more functionality:
https://github.com/theforeman/foreman/pull/5184/files#diff-1551b3b23e104a39b3f69746fe110eae